x-pack/platform/plugins/shared/dataset_quality/public/components/dataset_quality/table/columns.tsx

```diff
@@ -284,12 +284,16 @@ export const getDatasetQualityTableColumns = ({
     {
       name: (
         <EuiTableHeader data-test-subj="datasetQualityQualityColumn">
-          <EuiToolTip content={datasetQualityColumnTooltip}>
-            <span>
-              {`${datasetQualityColumnName} `}
-              <EuiIcon size="s" color="subdued" type="question" className="eui-alignTop" />
-            </span>
-          </EuiToolTip>
+          <span>
+            {`${datasetQualityColumnName} `}
+            <EuiIconTip
+              content={datasetQualityColumnTooltip}
+              type="question"
+              size="s"
+              color="subdued"
+              className="eui-alignTop"
+            />
+          </span>
```

Comment:
This changes a bit the actionable area, as now the tooltip will appear only on hover on the icon, while before it worked on the name `datasetQualityColumnName` label as well.

Reply:
Same for the other changes on this file